wordpress-mobile/WordPress-iOS
# Quick Start: add the last three tours

Refs #9447

This adds the three final tours for Quick Start.

### Publish a post 
![tour-post](https://user-images.githubusercontent.com/517257/46978840-cb4ece00-d084-11e8-8fd2-94f1a8033535.gif)

### Share your site
![tour-sharing](https://user-images.githubusercontent.com/517257/46978909-ed485080-d084-11e8-9b61-0d875dd16bf3.gif)

### Follow other sites
![tour-reader](https://user-images.githubusercontent.com/517257/46978948-07822e80-d085-11e8-95d9-46a7d17ca0d6.gif)


To test:
- Setup a site with Quick Start enabled (either create a new site or get my snippet)
- go through the three new tours
  - the followers tour should be resilient to leaving the reader tab in different states _before_ starting the tour. check it out!
- test on ipad, too

Do you mind doing this since you're already familiar with it, @etoledom?
